### PR TITLE
60 # Knex database model for Project created

### LINKED ISSUE #60: Create knex database model for projects

### PR DESCRIPTION
**Pull request by fructusmortus**
Resolved #60

### CODE REVIEW

File: module-3/back-end/src/models/project.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+const Model = (config) => {
+
+    const create = async project => {
+
+        return await knex(config.tableName).insert(project, [
+            'id',
+            'title',
+            'description',
+            'document_id',
```
Comment (shelby-carter) [inline module-3/back-end/src/models/project.js:16]: 'document_id' what is it for? If it's supposed to be a relationship between project and documents, I'd say it should read as plural `documents_id`, but either way I think it's better to have a separate table to store relationships between projects and documents, like `project_documents` for example and the structure of that table would be as follows
 - `id`
 - `document_id`
 - `project_id`

File: module-3/back-end/src/models/project.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+            'updated_at'
```
Comment (shelby-carter) [inline module-3/back-end/src/models/project.js:17]: Apart from having `updated_at` I think it's a good idea to also store `created_at` timestamp

Comment (fructusmortus) [reply module-3/back-end/src/models/project.js:17]: created_at will be added at the database level automatically at the moment of insert

File: module-3/back-end/src/models/project.js (hunk 1 above)
Comment (fructusmortus) [reply module-3/back-end/src/models/project.js]: I've changed structure

**Pull request merged**